### PR TITLE
Add network modification to hvdc line action

### PR DESCRIPTION
**Pull request by pjeanmarie**
**Please check if the PR fulfills these requirements**

- [ ] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem?**

**What kind of change does this PR introduce?**

feature

**What is the current behavior?**

HvdcLine.toModification raise not implemented exception.

**What is the new behavior (if this is a feature change)?**
Add a NetworkModification for HvdcLine and its extension HvdcAngleDroopActivePowerControl and implement toModification for a HvdcAction using this new NetworkModification.

**Does this PR introduce a breaking change or deprecate an API?**
- [ ] Yes
- [x] No

**If yes, please check if the following requirements are fulfilled**

- [ ] The *Breaking Change* or *Deprecated* label has been added
- [ ] The migration steps are described in the following section

**What changes might users need to make in their application due to this PR? (migration steps)**

**Other information**:

### CODE REVIEW

**Comment by pjeanmarie** [comment]
I have created a modification completly similar to the object HvdcAction (and sonar reports duplicate code). I can create 2 modifications one close to hvdc line object and another one for its extension.

**Comment by olperr1** [comment]
I would not split this network modification in 2.
Regarding the code duplication, are the getters necessary? A lot of network modifications don't have any.

**Review by olperr1 — CHANGES_REQUESTED** [review]

File: iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/HvdcLineModification.java [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+            return;
+        }
+        getActivePowerSetpoint().ifPresent(value -> hvdcLine.setActivePowerSetpoint((isRelativeValue() ? hvdcLine.getActivePowerSetpoint() : 0) + value));
+        getConverterMode().ifPresent(hvdcLine::setConvertersMode);
+        HvdcAngleDroopActivePowerControl hvdcAngleDroopActivePowerControl = hvdcLine.getExtension(HvdcAngleDroopActivePowerControl.class);
+        if (hvdcAngleDroopActivePowerControl != null) {
+            isAcEmulationEnabled().ifPresent(hvdcAngleDroopActivePowerControl::setEnabled);
+            getP0().ifPresent(value -> hvdcAngleDroopActivePowerControl.setP0((float) value));
+            getDroop().ifPresent(value -> hvdcAngleDroopActivePowerControl.setDroop((float) value));
+        }
```
[the remark this answers is not in the document]
Comment (pjeanmarie) [reply iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/HvdcLineModification.java]: done

**Comment by pjeanmarie** [comment]
> I would not split this network modification in 2. Regarding the code duplication, are the getters necessary? A lot of network modifications don't have any.

done

**Review by annetill — APPROVED** [review]

**Review by olperr1 — APPROVED** [review]

**Pull request merged**